### PR TITLE
feat(tests): reenable tests addressed by deis/controller #557

### PR DESCRIPTION
**Pull request by vdice**
Also updated some expectations to look for 503 (including still pending)
on account of https://github.com/deis/router/pull/153

### CODE REVIEW

**Comment by vdice** [comment]
[e2e build](https://ci.deis.io/job/workflow-test-pr/129/console) failing as expected until https://github.com/deis/controller/pull/557 is merged.

File: tests/builds_test.go [hunk 1]
```diff
@@ -140,9 +140,11 @@ var _ = Describe("Builds", func() {
...
 				Expect(len(scrapeProcs(testApp.Name, procsListing))).To(Equal(1))
 
-				// with routable 'cmd' process gone, curl should return StatusBadGateway
-				curlCmd = Cmd{CommandLineString: fmt.Sprintf(`curl -sL -w "%%{http_code}\\n" "%s" -o /dev/null`, testApp.URL)}
-				Eventually(cmdWithRetry(curlCmd, strconv.Itoa(http.StatusBadGateway), cmdRetryTimeout)).Should(BeTrue())
+				// TODO: still susceptible to intermittent curl timeouts
+				// (not returning in under 10 seconds)
+				// with routable 'cmd' process gone, curl should return StatusServiceUnavailable
+				//curlCmd = Cmd{CommandLineString: fmt.Sprintf(`curl -sL -w "%%{http_code}\\n" "%s" -o /dev/null`, testApp.URL)}
+				//Eventually(cmdWithRetry(curlCmd, strconv.Itoa(http.StatusServiceUnavailable), cmdRetryTimeout)).Should(BeTrue())
```
Comment (arschles) [inline tests/builds_test.go]: do we have an issue for this somewhere?

**Comment by arschles** [comment]
code LGTM. my comment at https://github.com/deis/workflow-e2e/pull/137#discussion_r58111751 doesn't block this PR

File: tests/builds_test.go (hunk 1 above)
Comment (vdice) [inline tests/builds_test.go]: I _think_ there was; will either find existing issue or create a new one to track this TODO.

Comment (vdice) [inline tests/builds_test.go]: https://github.com/deis/controller/issues/552 is the issue; will comment there mentioning to visit this TODO when issue resolved.

**Pull request merged**